cypress/fixtures/Suggested_Templates.json

```diff
@@ -42,7 +42,7 @@
 													{
 															"parameters": {},
 															"id": "551313bb-1e01-4133-9956-e6f09968f2ce",
-															"name": "When clicking \"Execute Workflow\"",
+															"name": "When clicking \"Test Workflow\"",
```

Comment:
why did you need to update the fixtures? the tests should have worked even with the old node names, right?

Reply:
why did they keep breaking? the node names of imported workflows should not change, right?

Reply:
Existing display names will not change! I only change it there because for some tests we were using this fixtures and asserting that the node display name was "Execute Node" instead "Test Node" so I changed it on all fixtures to avoid issues. I can revert the changes, and run the e2e tests again and only rename in the fixtures where we assert the new name.

Reply:
ah I see.. new nodes added in tests would need to be updated.. but not imported fixtures in tests.. it's your call here.. it's fine either way